index.js

```diff
@@ -59,6 +59,30 @@
 //. (for any type `a`) with an argument of type `a -> b` (for any type `b`),
 //. it returns a value of type `Maybe b`._
 //.
+//. ### Accessible pseudotype
+//.
+//. What is the type of values which support property access? In other words,
+//. what is the type of which every value except `null` and `undefined` is a
+//. member? [Object][] is close, but `Object.create(null)` produces a value
+//. which supports property access which is not a member of the Object type.
+//.
+//. Sanctuary uses the Accessible pseudotype to represent the set of values
+//. which support property access.
+//.
+//. ### TypeRep pseudotype
+//.
+//. What is the type of `Number`? One answer is `a -> Number`, since it's a
+//. function which takes an argument of any type and returns a Number value.
+//. When provided as the first argument to [`is`](#is), though, `Number` is
+//. really the value-level representative of the Number type.
+//.
+//. Sanctuary uses the TypeRep pseudotype to describe type representatives.
+//. For example:
+//.
+//.     Number :: TypeRep Number
+//.
+//. `Number` is the sole inhabitant of the TypeRep Number type.
+//.
```

Comment:
That sounds great! Two minor points alone

* "... `Number` is really the..." may be better said as "`Number` acts as the..."
* `TypeRep` can be called a "type function" or "type constructor" if you like instead of invoking the term "pseudotype". Taken together, `TypeRep` might also be called a family of types or a fiber bundle... though that last term is quite rare!

:+1: